python/conftest.py

```diff
@@ -0,0 +1,22 @@
+
+import pytest
+from prairielearn import QuestionData
+
+@pytest.fixture
+def question_data() -> QuestionData:
+    return {
+        "params": dict(),
```

Comment:
whats this for?

Reply:
This gets used by some of the new tests. Having an argument named `question_data` in a test function automatically creates a new dict by calling this function. See https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session